connect/src/web.ts

```diff
@@ -496,6 +547,82 @@ export class Ad4mConnectElement extends LitElement {
     this.loadFont();
   }
 
+  private async loginToHosting() {
+    try {
+    const response = await fetch('http://localhost:3000/login', {

```

Comment:
`localhost` in `loginToHosting()` seems wrong, or something only for local dev mode.

Reply:
yeah this needs to be updated with hosted API url